pallets/collator-selection/src/migrations.rs

```diff
@@ -0,0 +1,110 @@
+// Copyright 2020-2021 Manta Network.
+// This file is part of Manta.
+//
+// Manta is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Manta is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Manta.  If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use super::*;
+use frame_support::{
+	dispatch::GetStorageVersion,
+	pallet_prelude::Weight,
+	traits::{Get, PalletInfoAccess, StorageVersion},
+	Twox64Concat,
+};
+///  This migrates the pallet from the standard version by parity to our modified storage
+impl<T: Config> Pallet<T> {
+	pub fn migrate_to_v2() -> frame_support::weights::Weight {
+		use frame_support::migration::{
+			have_storage_value, remove_storage_prefix, storage_key_iter,
+		};
+		// Storage migrations should use storage versions for safety.
+		if Self::on_chain_storage_version() < 2 {
+			log::info!(" >>> Executing collator-selection V1->V2 migration!");
```

Comment:
@Dengjianping Wouldn't usefulness of that depend on us changing _all_ logs of the pallet to use the target flag?

Reply:
Only the upgrade struct in the calamari RT will be removed, the migration code here in the pallet can remain. I made the migrations module for that specific purpose.
- for future reference on migrations
- to keep a log of the migrations we applied to the chain

Reply:
@Dengjianping From the doc you linked:
> If you do not specify the logger target, it will be set to the crate's name 

So by omitting the target, `-lmanta-collator-selection=debug` sets this code's loglevel, which I think is better than providing an arbitrary string

Reply:
@Dengjianping please resolve if acceptable